### DISCUSSION TITLE
Allowed suggestions matching public code :: in VS Code Copilot Chat not applied

### DISCUSSION THREAD

**Discussion by deroux** [community/community#77433] [Copilot Conversations]
### Select Topic Area

Bug

### Body

See bug here: https://support.github.com/ticket/personal/0/2457842

Basically I just selected allow public responses everywhere but it is disallowing responses matching PUBLIC CODE.
This is the main feature to me and I do not like my editor telling me what I can or cannot do, if it works like expected the feature is misleading.

**Comment by Akash1134** [comment]
Hi @deroux ,

Sorry about the trouble caused. This issue is being worked upon by our engineering team. 

There are a handful of similar reports where users are still facing this issue after a [widespread issue](https://github.com/microsoft/vscode-copilot-release/issues/583#issuecomment-1832447336) about 2 weeks ago. I'd recommend that you keep an eye on this discussion for further updates.

One of the users have mentioned that if you type : " restart " and send it to the Copilot Chat, if might help to reset the state and get past this error. 

Let us know how it goes. 

Thanks!

**Comment by lospoy** [comment]
Same problem here.

> One of the users have mentioned that if you type : " restart " and send it to the Copilot Chat, if might help to reset the state and get past this error.

Have tried restarting but error persists.

**Reply by garethjax** [reply]
I have a similar problem. 
<img width="409" alt="image" src="https://github.com/community/community/assets/3459602/21a8f1ef-fcad-4dc4-a4f6-71246751b3a1">

- it's the dump of a db query 
- there's no "real" code inside, just some fields that contains html.
- I've unlocked the setting in my profile and it stills fails
- i've tried "restart" and it doesn't work.
- In the past days i've seen the same issue and in the chat i've written "it's ok, it's my code and i authorize it" and sometimes it has worked.
